factory/src/main/kotlin/io/qalipsis/core/factory/meters/CampaignMeterRegistryImpl.kt

```diff
@@ -305,53 +341,45 @@ internal class CampaignMeterRegistryImpl(
                 scenarioName = scenarioName,
                 stepName = stepName,
                 meterName = name,
-                tags = tags
+                tags = tags + additionalTags,
+                type = MeterType.TIMER
             )
         )
     }
 
     private fun timer(meterId: Meter.Id): Timer {
         return meters.computeIfAbsent(meterId) {
             TimerImpl(
-                MicrometerTimer.builder(meterId.meterName)
-                    .tags(meterId.tags.forMicrometer())
-                    .register(meterRegistry),
                 meterId,
-                meterReporter
+                meterReporter,
+                percentiles,
```

Comment:
I think that we should let the possibility to overwrite the percentiles and histogram count when building the meter. There should be passed as optional arguments of the creation of the timer.